File: benchmarks/random-1K/Main.java
public class Main {

    public static void main(String[] args) {
        long before = System.currentTimeMillis();

        double n = 0;
        for (int i = 0; i < 1000; i++) {
            n = Math.random();
        }

        long after = System.currentTimeMillis();

        System.out.println(after - before);
    }

}

File: benchmarks/random-1K/main.zig
const std = @import("std");

pub fn main() !void {
    const before: i128 = std.time.nanoTimestamp();

    const rand = std.crypto.random;
    var n: i32 = 0;
    for (0..999) |_| {
        n = rand.int(i32);
    }

    const after: i128 = std.time.nanoTimestamp();

    std.debug.print("{}", .{after - before});
}